### PR TITLE
DApp User can see Listings that are kept [ch634]

### PR DESCRIPTION
**Pull request by walfly**

### CODE REVIEW

**Comment by walfly** [comment]
still finishing actions part of it

File: packages/dapp/src/helpers/listingEvents.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Dispatch } from "redux";
+import { getTCR } from "./civilInstance";
+import { addListing } from "../actionCreators/listings";
+
+export async function initializeSubscriptions(dispatch: Dispatch): Promise<void> {
+  const tcr = getTCR();
+  await tcr.allApplicationsEver().forEach(async (listing) => {
```
Comment (nickreynolds) [inline packages/dapp/src/helpers/listingEvents.ts]: using allApplicationsEver is just a hack for now, right?

File: packages/dapp/src/components/Listings.tsx [hunk 2]
```diff
@@ -14,208 +13,100 @@ const StyledDiv = styled.div`
   color: black;
 `;
 
-export interface ListingsState {
-  applications: List<ListingWrapper>;
-  whitelistedListings: List<ListingWrapper>;
-  readyToWhitelistListings: List<ListingWrapper>;
-  inChallengeCommitListings: List<ListingWrapper>;
-  inChallengeRevealListings: List<ListingWrapper>;
-  canBeUpdatedListings: List<ListingWrapper>;
```
Comment (am9u) [inline packages/dapp/src/components/Listings.tsx]: Looks like the `canBeUpdatedListings` phase -- which I understand to be 'can Resolve Challenge' got dropped in this refactor. 

Is this no longer a valid phase? Is it wrapped into `awaitingAppealRequestListings`?

Comment (walfly) [reply packages/dapp/src/components/Listings.tsx]: Nothing was ever populating that list, so i killed it

**Pull request merged**